src/registry_schemas/schemas/amalgamation_application.json

```diff
@@ -52,16 +51,23 @@
                             "role": {
                                 "type": "string",
                                 "enum": [
+                                    "amalgamating",
                                     "holding",
-                                    "amalgamating"
+                                    "primary"
                                 ]
                             },
-                            "identifier": { "type": "string" },
-                            "foreignJurisdiction": { 
-                                "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/foreign_jurisdiction" 
+                            "identifier": {
+                                "type": "string"
                             },
-                            "legalName": { "type": "string" },
-                            "corpNumber": { "type": "string" }
+                            "foreignJurisdiction": {
+                                "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/foreign_jurisdiction"
+                            },
+                            "legalName": {
+                                "type": "string"
+                            },
+                            "corpNumber": {
```

Comment:
An _identifier_ is the corp number / registration number for a business.
Why are we using corpNumber?

Reply:
removed as discussed. will use identifier in both cases